text/0034-telemetry-policies.md

```diff
@@ -0,0 +1,111 @@
+- Cohort: Servers that are running for more than seven days with the new version of MS (> v0.10.1)
+- Cohort: Servers that are running for more than 30 days with the old version of MS (< v0.10.1)
+- Cohort: Servers that are running for more than 30 days with the new version of MS (> v0.10.1)
+- Graph: The total of every telemetry messages received
+- Graph: The total of every unique id seen
+- Graph: The number of servers that are running for more than 7 days/weekly
+- Graph: The number of servers that are running for more than 30 days/month
+- User Composition: Device composition
+- User Composition: Version composition
+- Indexes size distribution for more than seven days (<200k not taken into account)
+- Indexes size distribution for the others
+- Retention (not used)
+
+#### New Metrics
+
+##### Number of indexes per DB
+Discovery metric for federated search feature.
+
+##### Number of documents per indexes
+Discovery metric to know the average number of documents are stored in an index.
+
+##### Geographical server distribution, CPU server distribution, RAM server distribution, Disk server distribution
+Discovery metrics to choose more relevant data centers over the world for the SaaS cloud platform. Will also be used in order to perform price studies analysis.
+
+For the Geographical Server Distribution we need to make a call to a third-party like http://ip-api.com/json/113.14.168.85 to get `Country`, `City` and `Provider` at MeiliSearch launch.
+
+✅ Staying transparent is important to us, which is why the metrics mentioned in the previous sentence will be discussed on Github with our community.
+
+For the CPU, RAM and DISK metrics, we can use an internal system stat method to get this information.
+
+##### SDK distribution
+Discovery metric to check adoption of our new sdks and the use of existing ones
+
+##### Feature metrics
+E.g. We want to know more about feature usage like how many times the `ranking_rules` order are changed, which is the top most used `ranking_rules` in first position etc.. What is the average `response body` size, what is the average number of `filter` used for the search endpoint.
+
+❗️ Asking for new data points should be specified and explain why these data points will be used for in the feature specification.
+
+### V. Impact on documentation
+
+✅ Create a dedicated page aimed to explain why and how we collect anonymous data while being exhaustive on the metrics we collect. Related to https://github.com/meilisearch/documentation/issues/908.
+
+### VI. Impact on SDKs
+N/A
+
+### VII. Impact on CLI
+
+✅ Currently, MeiliSearch only say if `Anonymous telemetry` is enabled or not on the launch message. It should also provide a message explaing in a few words that we are collecting anonymized MeiliSearch behavior metrics to enhance the product for future releases. Thus, displaying a link to the analytics documentation page.
```

Comment:
What is the exact message so that I can open the issue?

Reply:
@curquiza I've updated the specification with a message!